### DISCUSSION TITLE
Is skywalking can collector application inner invoke link road like pinpoint？

### DISCUSSION THREAD

**Discussion by feixiangshenhua** [apache/skywalking#12576] [General]
like this，You can clearly see the in-app call link。
![image](https://github.com/user-attachments/assets/0e6b3981-5093-4955-83c1-753d15200a1e)

but our skywalking don't have this function,like this
![image](https://github.com/user-attachments/assets/c6eac830-580b-4b37-a8cb-7eb1cb3c24d9)

**Comment by wu-sheng** [comment]
That depends on which plugins activated. There is no concept like you said(inner invoke).
